### ISSUE TITLE
Missing intern_clip_feat

### ISSUE THREAD

**Issue by Tanveer81** [huangb23/VTimeLLM#22]
Are there missing stage2 training data? 

No such file or directory: 'ROOT/intern_clip_feat/aMEDuIHmJJM.npy'

**Comment by SaMMyCHoo** [comment]
Same error there, and there are also some more missing feats. But the code is still working~

**Comment by huangb23** [comment]
Yes. For a small subset of videos in the dataset, our feature extraction process failed. This won't cause critical errors; our checkpoint was trained in a similar manner.
